src/keyboard.cpp

```diff
@@ -123,11 +123,38 @@ void keyboard::show()
 	QWidget::show();
 	//Moving after showing because window height isn't set until show()
 	this->move(0,QApplication::desktop()->screenGeometry().height() - height());
+    QTimer::singleShot(1, this, SLOT(selectAllInFocusedWidget()));
```

Comment:
I'd take a look at the settings on your editor and select the 'hard tabs' option. Our enforcement of code style isn't great, but let's at least try to maintain consistency.